packages/app-check/src/recaptcha.ts

```diff
@@ -143,11 +143,19 @@ function renderInvisibleWidget(
   grecaptcha: GreCAPTCHA,
   container: string
 ): void {
+  const state = getStateReference(app);
```

Comment:
Is this needed?

Reply:
It's referenced in 160. Apparently it used to be under the widgetId block, I should put it back, it's weird here.